index.html

```diff
@@ -1,19 +1,91 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <title>Karma</title>
     <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
     <link rel="stylesheet" href="css/normalize.css">
     <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico">
 </head>
+
 <body>
 
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
 
+
+    <!-- header part -->
+    <header>
+        <img class="karmaIcon" src="./img/karma-logo.svg" alt="karma's icon">
+        <nav>
+            <ul>
+                <li>
+                    Meet Karma
+                </li>
+                <li>
+                    How it Works
+                </li>
+                <li>
+                    Store
+                </li>
+                <li>
+                    Blog
+                </li>
+                <li>
+                    Help
+                </li>
+                <li>
+                    Login
+                </li>
+            </ul>
+        </nav>
```

Comment:
It is suggested to use `a`  tag for each nav item as they will be used as links.
you can put  `#` for now or a related section of the page. For logo `img` you can navigate to home page as a best practice and better UX
```
<li>
      <a href="#">Meet Karma </a>
 </li>
 ```